pkg/cloudfoundry/container_tagger.go

```diff
@@ -0,0 +1,108 @@
+package cloudfoundry
```

Comment:
This will need a copyright header.  Also, this new `pkg/` should probably be added to CODEOWNERS (it's defaulting to my team right now).

It'd be great if the package had a docstring explaining what it is.  Why does this need a top-level `pkg`?  Why not `pkg/util/cloudfoundry`?  Will this only ever contain the container tagger?  If so, should it be `pkg/cfcontainerfactory` or something?  Or maybe `pkg/cloudfoundry/containertagger`?

Reply:
One of the reasons why I didn't put it in the `pkg/util/cloudproviders/cloudfoundry` (if that's what you meant) was that it produced a circular dependency with `pkg/metadata/host` which imports `pkg/util/cloudproviders`.

Otherwise it'll be more confusing to have both `pkg/util/cloudfoundry` and `pkg/util/cloudproviders/cloudfoundry`.

I'm on it for the file changes. 

Thanks!

Reply:
Yeah, this is a super-annoying weakness of Go :(